build.gradle.kts

```diff
@@ -196,6 +198,21 @@ subprojects {
     }
   }
 
+  if (project.name != "meta") {
+    apply(plugin = "net.ltgt.errorprone")
+    dependencies {
+      errorprone("com.google.errorprone:error_prone_core:2.10.0")
+    }
+
+    tasks.withType<JavaCompile>().configureEach {
+      options.errorprone.isEnabled.set(true)
+      options.errorprone.disableAllChecks.set(true)
+      options.errorprone.enable(
+        "AnnotateFormatMethod"
```

Comment:
Only enable `@FormatMethod`, because we enable  `-Werror`, Or there are a lot of warnings that are treated as an error.

Reply:
good improvement！I had the same problem last night.